test/e2e/util.go

```diff
@@ -167,10 +183,12 @@ func waitForPodsRunningReady(ns string, minPods int, timeout time.Duration) erro
 			Logf("Error getting pods in namespace '%s': %v", ns, err)
 			continue
 		}
-		nOk := 0
+		nOk, badPods := 0, []api.Pod{}
 		for _, pod := range podList.Items {
 			if res, err := podRunningReady(&pod); res && err == nil {
 				nOk++
+			} else {
+				badPods = append(badPods, pod)
 			}
 		}
 		Logf("%d / %d pods in namespace '%s' are running and ready (%v elapsed)",
```

Comment:
While you're cleaning up, could you round this time to tenths of a second or something more reasonable to humans? :)